dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/filter/WebAppFilterManager.java

```diff
@@ -138,7 +138,6 @@ public boolean removeEldestEntry(Map.Entry eldest) {
     com.ibm.ws.webcontainer.webapp.WebApp webApp;
 
     private ICollaboratorHelper collabHelper;
-    private IWebAppNameSpaceCollaborator webAppNameSpaceCollab;
```

Comment:
@jhanders34 , won't this leave an unused import `import com.ibm.wsspi.webcontainer.collaborator.IWebAppNameSpaceCollaborator;`? Can we clean that up as well?

Reply:
Done